app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt

```diff
@@ -46,8 +46,13 @@ class TracingInitializer : Initializer<Unit> {
                 writesToFilesConfiguration = WriteToFilesConfiguration.Disabled
             )
         } else {
+            val config = if (BuildConfig.BUILD_TYPE == "nightly") {
+                TracingFilterConfigurations.nightly
+            } else {
+                TracingFilterConfigurations.release
+            }
             TracingConfiguration(
-                filterConfiguration = TracingFilterConfigurations.release,
+                filterConfiguration = config,
                 writesToLogcat = false,
                 writesToFilesConfiguration = WriteToFilesConfiguration.Enabled(
```

Comment:
Ah I was wondering why I did not see any files when working on https://github.com/element-hq/element-x-android/pull/2283.

Writing into files is only enabled on release and nightly builds 🤦

Reply:
Ooops. Maybe we should change that?

Reply:
Actually it makes sense, since we do not really send rageshake when using a debug version (we read directly on logcat). It may be an issue for APK installed from Diawi (debug version), but we always have the logcat file so I think it's enough.